### ISSUE TITLE
unity_pcl not found when loading PCD files

### ISSUE THREAD

**Issue by ROS-lover** [autocore-ai/MapToolbox#51]
Hey guys,

Found this issue on ubuntu 18 and 2020 and 2019.4 unity versions. As a follow up question: is there a way I could use the old version of this toolbox as it used to work for me? 

Thanks!

![image](https://user-images.githubusercontent.com/51894258/94611820-23be0c80-0257-11eb-89a0-c21bb6202f36.png)

**Comment by xqrkushnir** [comment]
Hi there,
You can pull a specific commit, branch, or tag as written here: https://docs.unity3d.com/Manual/upm-git.html (under `Specifying a package in a subfolder of the repository`).

I do confirm that I get the same issue for Ubuntu 18 and Unity 2019.4 LTS. I also can import PCD flawlessly in Windows 10 and Unity 2020.1.7f1.

**Comment by alperenbayraktar** [comment]
Well, i fixed it by adding Windows build support to unity version i used.

**Comment by xqrkushnir** [comment]
> Well, i fixed it by adding Windows build support to unity version i used.

Didn't do the trick for me. What version of Unity/Ubuntu do you have?

**Comment by alperenbayraktar** [comment]
> 
> 
> > Well, i fixed it by adding Windows build support to unity version i used.
> 
> Didn't do the trick for me. What version of Unity/Ubuntu do you have?

Windows 10 Unity 2019.3.3f1

**Comment by auronman** [comment]
HI, i have the same issue. DId you solve it in Ubuntu?

**Comment by xqrkushnir** [comment]
> HI, i have the same issue. DId you solve it in Ubuntu?

Not really. I've imported a PCD map in Windows, then moved this project on Ubuntu
